gradle.properties

```diff
@@ -16,4 +16,5 @@
 # This option should only be used with decoupled projects. More details, visit
 # http://www.gradle.org/docs/current/userguide/multi_project_builds.html#sec:decoupled_projects
 # org.gradle.parallel=true
-android.enableUnitTestBinaryResources=true
\ No newline at end of file
+android.enableJetifier=true
```

Comment:
this will remain `true` until we reach the end of the project and try to turn it off.